### PR TITLE
Testing features

### PR DESCRIPTION
**Pull request by edwinmoradian90**
**IMPORTANT** I was on leave for several months and was given permission to finish the entire javascript section alone.

### CODE REVIEW

File: functions.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
+const functions = {
+    capitialize: (string) => string[0].toUpperCase() + string.slice(1),
```
Comment (mcrd25) [inline functions.js]: [OPTIONAL]
typo, should be `capitalize`

**Comment by edwinmoradian90** [comment]
# Summary of changes made
- Added stickler and eslintrc
- Updated errors in code accordingly

**Comment by edwinmoradian90** [comment]
Thanks!

**Comment by edwinmoradian90** [comment]
Hm status is still on “changes required”

**Comment by edwinmoradian90** [comment]
## IMPORTANT

The project was already approved but on my dashboard, I don't have 'submit project' flag, so I'm sending out for reapproval. Thanks.

**Pull request merged**